src/app/users/store/users.actions.ts

```diff
@@ -3,16 +3,21 @@ import { IUser } from '../user.interface';
 
 const begin = createAction('[Users] Get All - Begin');
 const success = createAction(
-  '[Users] Get All - Success',
+  '[Users Effect] Get All - Success',
```

Comment:
Does not get sense form me this name 
[Users Effect]

The idea of this naming convention is to set the name of the module there.
[Users]

Reply:
Thanks Diego, I was following a naming convention seen on a video. Changed right now